protocol/app/upgrades/v6.0.0/upgrade_container_test.go

```diff
@@ -1,25 +1,23 @@
-//go:build all || container_test
```

Comment:
this needs to be put back. These are build constraints that tells go to only include this file in the build if either the `all` tag or `container_test` tag is specified when invoking the Go build or test commands. (e.g.`go test -tags='container_test'`, which we specify in `make test-container`) 

You can remove this line locally to run only these tests, but we shouldn't check this change in

Reply:
ah my b - forgot to add back